mathesar/database/types/constants.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+TYPE_SCHEMA = "mathesar_types"
```

Comment:
I think you should use the `APP_PREFIX` from `base` instead of `mathesar_` here.

Reply:
Can't this just go in the `types.py` file? Or maybe in `__init__.py`?

Reply:
I think I'll put it in the `__init__.py` .

As for the `APP_PREFIX`, the way that `SQLAlchemy` handles quoting for user defined functions results in problems if the schema needs quoting (as our `APP_PREFIX` does).  I'm hoping to find a way around that, but if I can't it seemed preferable to have the schema name defined near where its used so that the problem isn't spread around (the problem being the need for a no-quotes-needed schema name)

Reply:
I ended up keeping the constants file (I expect it to grow), but managed to get `APP_PREFIX` working